storage/src/main/java/io/delta/storage/internal/LogStoreErrors.java

```diff
@@ -31,8 +29,7 @@ public static boolean isNonFatal(Throwable t) {
         if (t instanceof VirtualMachineError ||
             t instanceof ThreadDeath ||
             t instanceof InterruptedException ||
-            t instanceof LinkageError ||
-            t instanceof ControlThrowable) {
+            t instanceof LinkageError) {
```

Comment:
whats this change

Reply:
this is a java library .. we can't import a scala error

Reply:
ahhaa... how was it  not failing till now? w